lib/index.d.ts

```diff
@@ -5,6 +5,8 @@
 					/**
 					 * Gets the current test. Added by cypress-plugin-retries
 					 */
-					currentTest: Mocha.ISuiteCallbackContext
+					currentTest: Mocha.ISuiteCallbackContext & {
```

Comment:
Can be changed to `Mocha.Test`

Reply:
Not that much into mocha, should have checked that, will crosscheck and update the MR.
Thank you!